.gitignore

```diff
@@ -149,3 +149,4 @@ cython_debug/
 
 # Mac OS
 .DS_Store
+exec.py
```

Comment:
that's a custom file which might have different usages/names on different machines, so it shouldn't go into the .gitignore